src/components/Button/Button.tsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+
+interface ButtonProps extends React.ButtonHTMLAttributes<HTMLButtonElement> {
+  children: React.ReactNode;
+  name: string;
+}
+
+const Button: React.FC<ButtonProps> = ({ children, name, ...props }) => {
+  return (
+    <>
+      <button
+        type="button"
+        name={name}
```

Comment:
No need to manually include the `name` prop here, it is already included as an optional prop in the `{...props}` that we spread here.